addons/l10n_it_edi_withholding/__init__.py

```diff
@@ -1,21 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-import logging
-from . import models
-
-_logger = logging.getLogger(__name__)
-
-def _l10n_it_edi_withholding_post_init(env):
-    """ Existing companies that have the Italian Chart of Accounts set """
-    for company in env['res.company'].search([('chart_template', '=', 'it') , ('parent_id', '=', False)]):
-        _logger.info("Company %s already has the Italian localization installed, updating...", company.name)
-        ChartTemplate = env['account.chart.template'].with_company(company)
-        ChartTemplate._load_data({
```

Comment:
Didn't test, but is it ok to remove?
Is the additional data loaded correctly in all the cases?